examples/aggregator/module/rust/src/data.rs

```diff
@@ -60,15 +60,18 @@ impl From<SparseVector> for SerializedSparseVector {
 impl Monoid for SparseVector {
     fn identity() -> Self {
         SparseVector {
-            map: HashMap::new(),
+            entries: HashMap::new(),
         }
     }
 
     /// Combines two Sparse Vectors by adding up values corresponding to the same keys.
     fn combine(&self, other: &Self) -> Self {
-        other.map.iter().fold(self.clone(), |mut svec, (&i, &v)| {
-            *svec.map.entry(i).or_insert(v) += v;
-            svec
-        })
+        other
+            .entries
+            .iter()
+            .fold(self.clone(), |mut svec, (&i, &v)| {
+                *svec.entries.entry(i).or_insert(v) += v;
```

Comment:
Doesn't this cause v to be calculated twice, if the entry was empty? I expect it should be

```suggestion
                *svec.entries.entry(i).or_insert(0) += v;
```

BTW, perhaps some unit tests would be useful here :)